packages/core/content-releases/server/src/services/__tests__/release.test.ts

```diff
@@ -41,6 +42,26 @@ describe('release service', () => {
         'No release found for id 1'
       );
     });
+
+    it('throws an error if the release is already published', () => {
+      const strapiMock = {
+        ...baseStrapiMock,
+        entityService: {
+          findOne: jest.fn().mockReturnValue({ id: 1, name: 'test', releasedAt: new Date() }),
+        },
+      };
+
+      // @ts-expect-error Ignore missing properties
+      const releaseService = createReleaseService({ strapi: strapiMock });
+
+      const mockReleaseArgs = {
+        name: 'Release name',
+      };
+
+      expect(() => releaseService.update(1, mockReleaseArgs, { user: mockUser })).rejects.toThrow(
```

Comment:
Shouldn't you test updating a release, adding a release action to a release, deleting a release action from a release, and updating a release action in a release?